### PR TITLE
Add v2.4.0 of mattermost-plugin-jira to the Marketplace

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
This ticket is a request to add Jira v2.4.0 to the marketplace!

#### Ticket Link
n/a

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -3701,6 +3812,49 @@
...
+            "type": "bool",
+            "help_text": "When true, a [user satisfaction survey](!https://mattermost.com/pl/default-nps) will be sent to all users quarterly. The survey results will be used by Mattermost, Inc. to improve the quality and user experience of the product. Please refer to our [privacy policy](!https://about.mattermost.com/default-privacy-policy) for more information on the collection and use of information received through our services.",
+            "placeholder": "",
+            "default": true
+          }
+        ]
+      }
+    },
+    "updated_at": "2020-04-23T17:52:07Z"
+  },
```
Comment (hanzei) [inline plugins.json:3857]: That is part of https://github.com/mattermost/mattermost-marketplace/pull/63

File: plugins.json [hunk 2]
```diff
@@ -3940,44 +4094,5 @@
...
-      "webapp": {
-        "bundle_path": "webapp/dist/main.js"
-      },
-      "settings_schema": {
-        "header": "",
-        "footer": "",
-        "settings": []
-      }
-    },
-    "updated_at": "2020-04-22T18:12:27.541293964Z"
```
Comment (hanzei) [inline plugins.json:3981]: `make plugins.json` should not drop existing pluings. I filled https://github.com/mattermost/mattermost-marketplace/issues/67 to track that bug. For now please manually add it back.

**Comment by DHaussermann** [comment]
Tested for and passed from `add_mattermost-plugin-jira_v2.4.0` branch
- Version number and icon are correct
- Tested upgrade from prev and well as fresh install
No issues found. LGTM!
@jfrerich  please ping me if another round of testing is needed based on comment from @hanzei above.

**Comment by DHaussermann** [comment]
Tested again after last commit LGTM!

**Pull request merged**